sphinx_scality/static/css/theme.css

```diff
@@ -3,3 +3,122 @@
 
 @import url("./fonts.css");
 @import url("./current.css");
+
```

Comment:
Those styles should not be in `theme.css`, which is just an entrypoint for the others. If you want to split header from footer and the likes, create new files to hold those styles, and keep things neat. Otherwise, we'll never know where the styles for the header are since they are disseminated all over the place.